metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -581,6 +581,9 @@ def post_entity(
         curl_command,
     )
     response = session.post(url, payload)
+    response_json = response.json()
+    if 'message' in response_json:
```

Comment:
you can use `if not response.ok` instead, so you don't have to rely on the structure of the message.

Reply:
This may be safer^